lib/node_modules/@stdlib/math/base/special/fast/alpha-max-plus-beta-min/benchmark/c/native/benchmark.c

```diff
@@ -0,0 +1,138 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+/**
+* Benchmark `hypot`.
+*/
+#include "stdlib/math/base/special/fast/alpha-max-plus-beta-min.h"
```

Comment:
```suggestion
#include "stdlib/math/base/special/fast/alpha_max_plus_beta_min.h"
```

We do follow this type of convetion, please change the file name as well.

Reply:
@USERSATOSHI You did not update the include path. We use snakecase, not kebabcase, for file names.

Reply:
updated to snakecase